src/remote-config/remote-config.ts

```diff
@@ -176,16 +176,6 @@ export class RemoteConfig {
     return new RemoteConfigTemplateImpl(template);
   }
 
-  /**
-   * Instantiates {@link RemoteConfigServerTemplate} and then fetches and caches the latest
-   * template version of the project.
-   */
-  public async getServerTemplate(options?: RemoteConfigServerTemplateOptions): Promise<RemoteConfigServerTemplate> {
-    const template = this.initServerTemplate(options);
-    await template.load();
-    return template;
-  }
-
```

Comment:
do we need to update the internal API proposals to reflect this change?

Reply:
I was thinking this is a subset of the approved API, ie I'm not adding an unreviewed method, so it wasn't worth a full amendment, but I'll add a note to the doc, comment-in the approvers and see if they have any objections.

Reply:
Amendment approved!